examples/systemd/post-install

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# this post upgrade script is run each time the teleport package is upgraded
+
+set -eu
+
+case "$1" in
+    configure)
+        # Define old paths and new base directory.
+        OLD_PATHS=(
+            "/usr/local/bin/teleport"
+            "/usr/local/bin/tsh"
+            "/usr/local/bin/tctl"
+            "/usr/local/bin/tbot"
+        )
+        NEW_BASE_DIR="/usr/local/teleport-system/bin"
+
+        echo "Cleaning up old files and creating symlinks where necessary..."
+
+        for FILE in "${OLD_PATHS[@]}"; do
+            BASENAME=$(basename "$FILE")
+            NEW_PATH="$NEW_BASE_DIR/$BASENAME"
+
+            if [ -e "$FILE" ] && [ ! -L "$FILE" ]; then
+                echo "Removing old binary at $FILE..."
```

Comment:
Curious why we'd want to remove binaries here? `teleport-update link` will fail in this case. Should we change that behavior?